### PR TITLE
Re-enable support for Go 1.12

### PR DESCRIPTION
**Pull request by gj**
PR checklist:

- [x] Added changelog entry.

### CODE REVIEW

File: languages/go/internal/host/host.go [hunk 1]
```diff
@@ -329,13 +329,7 @@ func (h Host) ToPolar(v interface{}) (*Value, error) {
...
 			return nil, err
 		}
-		var repr string
-		value := reflect.ValueOf(v)
-		if value.IsZero() {
-			repr = fmt.Sprintf("%v{}", value.Type())
-		} else {
-			repr = fmt.Sprintf("%v{%v}", value.Type(), v)
-		}
+		repr := fmt.Sprintf("%T%+v", v, v)
```
Comment (gj) [inline languages/go/internal/host/host.go:332]: Alternative to `%T%+v` is `%#v`, but I slightly preferred the formatting this way.

### `%T%+v`

```
ZeroFoo{ZeroString: ZeroNumber:0}
Foo{String:hi Number:42}
```

### `%#v`

```
ZeroFoo{ZeroString:\"\" ZeroNumber:0}
Foo{String:\"hi\" Number:42}
```

File: languages/go/tests/unit_test.go [hunk 2]
```diff
@@ -300,6 +302,38 @@ func TestRuleTypes(t *testing.T) {
...
+
+	polarValue, err = host.ToPolar(Foo{Name: "Zooey", Num: 42})
+	if err != nil {
+		t.Fatalf("host.ToPolar failed: %v", err)
+	}
+	switch variant := polarValue.ValueVariant.(type) {
+	case ValueExternalInstance:
+		expected := "oso_test.Foo{Name:Zooey Num:42}"
+		if *variant.Repr != expected {
+			t.Errorf("Zero-value Foo{} repr didn't match.\n\tExpected: %v\n\tReceived: %#v", expected, *variant.Repr)
```
Comment (samscott89) [inline languages/go/tests/unit_test.go:330]: Super nit: error message is wrong. This is not the zero value case

**Review by samscott89 — APPROVED** [review]
Looks good. Please update the test message if you can.

File: languages/go/tests/unit_test.go (hunk 2 above)
Comment (gj) [reply languages/go/tests/unit_test.go:330]: A _super nit_??? This calls for _Super Nit Man_!!!

Comment (gj) [reply languages/go/tests/unit_test.go]: 772c072

**Pull request merged**